cypressFetchMock.cjs

```
@@ -0,0 +1,246 @@
+  }
+
+  return null
+}
+
+/**
+ * Patches `globalThis.fetch` for the lifetime of the Node process (dev server).
+ * Idempotent enough for instrumentation: repeated calls would stack wrappers, so
+ * only invoke once from `instrumentation.js`.
+ */
+module.exports = function setupCypressFetchMock() {
+  if (typeof globalThis.fetch !== 'function') return
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)